### ISSUE TITLE
Settlements: Leave impediments in City Planner

### ISSUE THREAD

**Issue by JefferyWood** [mainIine/foe-helfer-extension#2741]
I love the "Remove Buildings" feature in the City Planner, but when I do that in a settlement, it also removes the impediments. So I either have to replace the impediments or remove everything one by one.

Could City Planner be updated to leave the impediments in place when Remove Buildings is clicked?

**Comment by panda667** [comment]
Yes please! I can't smoothly plan my egyptian settlement because the impediments keep moving and it takes too long to manually put them all back.
